### PR TITLE
Add EvolvConfig test

### PR DESCRIPTION
**Pull request by dvoineu**

### CODE REVIEW

File: EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/genome.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+    "web": {
```
Comment (robertsevernsentient) [inline EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/genome.json:2]: I would prefer if the keys were renamed so this didn't give the impression the values used for a web experiment were relevant here -- the only extra complexity is that the key is returning an object

File: EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/audience_query.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+[
```
Comment (robertsevernsentient) [inline EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/audience_query.json:1]: Isn't this an allocation? Not specifically an audience query?

**Review by robertsevernsentient — APPROVED** [review]

File: EvolvSwiftSDK/EvolvSwiftSDK/Sources/Core/EvolvContextImpl.swift [hunk 3]
```diff
@@ -22,26 +22,28 @@ import Foundation
 public class EvolvContextImpl: EvolvContext {
     
     private var uid: String
+    private var sid: String?
```
[the remark this answers is not in the document]
Comment (dvoineu) [reply EvolvSwiftSDK/EvolvSwiftSDK/Sources/Core/EvolvContextImpl.swift:25]: that's why it was optional in the code. I've just forgot to remove it.

File: EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/audience_query.json (hunk 2 above)
Comment (dvoineu) [reply EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/audience_query.json:1]: Exactly. Will be removed

File: EvolvSwiftSDK/EvolvSwiftSDK/Sources/Core/EvolvConfigImpl.swift [hunk 4]
```diff
@@ -24,9 +24,9 @@ struct EvolvConfigImpl: EvolvConfig {
     
     public enum Default {
         public static let httpScheme: String = HttpConfig.httpScheme
-        public static let domain: String = HttpConfig.domain
+        public static let domain: String = HttpConfig.devDomain
```
[the remark this answers is not in the document]
Comment (dvoineu) [reply EvolvSwiftSDK/EvolvSwiftSDK/Sources/Core/EvolvConfigImpl.swift:27]: accepted

File: EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/genome.json (hunk 1 above)
Comment (dvoineu) [reply EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/genome.json:2]: You mean to change key with name "web" to something different when parsing JSON and creating Model with a new name field (property) accordingly?

Comment (robertsevernsentient) [reply EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/genome.json:2]: No, I just mean changing your example genome to use some names that are more general and not tuned for web experiments. You grabbed this genome to get a more complex genome to read in and test. Really all we're looking for is the ability to have the variant value be an object isn't it?

Comment (dvoineu) [reply EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/genome.json:2]: Ok, understood

**Pull request merged**